pkgs/applications/misc/archivebox/default.nix

```
@@ -34,38 +40,47 @@ let
           rev = "e43f383dae3a35237e42f6acfe1207a8e7e7bdf5";
           hash = "sha256-NAMa78KhAuoJfp0Cb0Codz84sRfRQ1JhSLNYRI4GBPM=";
         };
+        # possibly a real issue, but that version is not supported anymore
+        disabledTests = [ "test_should_highlight_bash_syntax_without_name" ];
       });
     };
   };
 in
 
 python.pkgs.buildPythonApplication rec {
   pname = "archivebox";
-  version = "0.6.2";
+  version = "0.7.1";
 
```

```suggestion
pyproject = true;

```
Usually you'll have to set `nativeBuildInputs`.

---

To anything specific? The project seems to work as-is.

---

pdm-backend shouldn't be propagated.